repo/packages/R/riak/2/package.json

```diff
@@ -9,8 +9,8 @@
   "scm": "https://github.com/basho-labs/riak-mesos.git",
   "website": "http://basho-labs.github.io/riak-mesos/",
   "postInstallNotes": "Thank you for installing Riak on Mesos. Visit https://github.com/basho-labs/riak-mesos for usage information.",
-  "preInstallNotes": "The Riak framework should have at least 1GB of RAM and 0.5 CPUs to perform successfully.",
-  "postUninstallNotes": "The Mesos Riak Framework has been uninstalled, .",
+  "preInstallNotes": "The Riak framework should have at least 16GB of RAM and 8.0 CPUs to perform successfully. Documentation: https://github.com/dcos/examples/tree/master/1.8/riak",
```

Comment:
@platon, @travisbhartwell Should say that each node in Riak Mesos Framework should have at least 16GB and 8 vCPUs, and we recommend 32GB of RAM and 32 vCPUs for production environments. For development and testing users can use fewer HW resources, understanding the performance will be affected.